app/views/comments/edit.html.erb

```diff
@@ -0,0 +1,7 @@
+<a href="/articles/<%= @article.id %>">
+  << Back
+</a>
+
+<h1>Edit Comment</h1>
+
+<%= render "form", article: @article, comment: @article.comments %>
```

Comment:
for call partial `form` to edit here, we need to send only one `comment` not `comments`, so this could change to
```
<%= render "form", article: @article, comment: @comment %>
```